### ISSUE TITLE
branding direction

### ISSUE THREAD

**Issue by marykatefain** [HarvestHub/GardenHub#9]
Some things that would be helpful (not a priority, but just putting it on your radar):

1. Brand colors
2. Logo
3. Brand fonts preferences
4. Images you would like used (ex as background image)

If you don't have any of these picked out- that's okay! We can help you choose them. For example we are currently just running with a Creative Commons pic we found online and fonts we just personally like.

**Comment by alexgleason** [comment]
I'm liking the purple and green theme we've sporadically adopted.

For fun, I spent $5 on Fiverr to have someone develop a GardenHub logo for us. We probably wont use it but I'm curious to see what this guy comes up with.

**Comment by alexgleason** [comment]
Breaking!!! Our $5 logo by d_design of Bulgaria came through. What do you think?

![fo1442882a72](https://user-images.githubusercontent.com/3639540/32681514-f0780cca-c63d-11e7-8ce2-f3fe6c987514.jpg)

**Comment by marykatefain** [comment]
This is bad. xD

**Comment by alexgleason** [comment]
I think the G can be salvaged and used.

**Comment by alexgleason** [comment]
Here are a couple of ideas with the G.

![gardenhub3](https://user-images.githubusercontent.com/3639540/32685667-d7160e00-c663-11e7-9726-537e6f2f5f60.png)

^ Less tall

![gardenhub2](https://user-images.githubusercontent.com/3639540/32685668-d732d724-c663-11e7-9625-a9d27c109ee8.png)

^ The H is built into the G

I'm also keen to try more vendors on Fiverr for more ideas.

**Comment by alexgleason** [comment]
I submitted a request for 4 additional vendors on Fiverr. We'll have a nice variety of options to choose from.

**Comment by alexgleason** [comment]
We got 3 more logos. Still waiting for 2 more.
![3](https://user-images.githubusercontent.com/3639540/32700895-a4f8bde6-c799-11e7-98f7-9e9e91a526f5.png)
![1](https://user-images.githubusercontent.com/3639540/32700894-9fa9bcaa-c799-11e7-9f0e-28b55f9a16f2.png)
![gardenhug01](https://user-images.githubusercontent.com/3639540/32700890-94af9892-c799-11e7-9688-ffdb65bc526f.png)

**Comment by alexgleason** [comment]
We got another slew of logos. I'd rather they try to make 1 good logo than 4 bad logos, but... here we are nontheless. Now we're waiting on just one more person.

![4](https://user-images.githubusercontent.com/3639540/32701963-2262b8e6-c7ad-11e7-878b-e7567e0796b6.jpg)
![3](https://user-images.githubusercontent.com/3639540/32701964-22741820-c7ad-11e7-9cc5-ee0c78786767.jpg)
![2](https://user-images.githubusercontent.com/3639540/32701965-228149c8-c7ad-11e7-8bcc-46fa981e4e5f.jpg)
![1](https://user-images.githubusercontent.com/3639540/32701966-228b49fa-c7ad-11e7-8789-83aa9dae558e.jpg)

Unless the last person gives us something really good, I think this one is the winner:

![gardenhub](https://user-images.githubusercontent.com/3639540/32702096-6161dc14-c7af-11e7-831e-7dc067ad82f6.png)

**Comment by alexgleason** [comment]
We got another one, but I was mistaken; there's still one more on the way after this.

![kretek1936](https://user-images.githubusercontent.com/3639540/32705096-05fd41f4-c7de-11e7-9ca3-aa55aa1708f8.png)

**Comment by alexgleason** [comment]
Okay, last three. This was a fun experiment.

![gardenhub-logo-c](https://user-images.githubusercontent.com/3639540/32738630-90864f40-c86b-11e7-8d35-8c76f3e15671.png)
![gardenhub-logo-b](https://user-images.githubusercontent.com/3639540/32738631-90a14638-c86b-11e7-9ddb-38ddbe789d60.png)
![gardenhub-logo-a](https://user-images.githubusercontent.com/3639540/32738633-90d47abc-c86b-11e7-8c87-975d2fa2dd7c.png)

I still feel pretty strongly that this one is our winner:

![](https://user-images.githubusercontent.com/3639540/32702096-6161dc14-c7af-11e7-831e-7dc067ad82f6.png)

**Comment by JenniferGold** [comment]
Alex and Mary Kate,

A couple of thoughts re: colors and logos:

1.  this was a superfun thread to read.

2.  I like the green and purple color theme generally.  I have a happy
weakness for the super bright green of the first logo (you should see our
park potty shed whose similarly-bright-green door was my uber-enthusiastic
choice; great story there; always a great story).

3.  I like the favorite logo for now, although I reserve the right to get a
better one later.  Ideally, what I would love is a very simple, strong logo
that somehow combines images/lines of the three key dimensions of our
projects:  growing food (green plant/leaf, bright candy-apple red tomato,
and/or plant with roots in ground?), people (harvest gardener workers,
recipients of donated food, someday the hyper-local purchasers of food?),
and an urban environment (buildings?).  See attached PPT mostly for its PPT
template (with just clip art, publicly available images) that includes
images of food, people, and urban locale (you can ignore the old gardenhub
sceen shots!  hahaha.).

how are you guys anyway?  how the house collective???  who's making all the
dinners lately?  do you have house chores in the app as well?

Jen

**Comment by JenniferGold** [comment]
p.s.  just to clarify - the PPT I last sent, with its slide template logo
thing is not the right colors, and it is too complex, and a bit too much,
but I just wanted to share its ideas re: image shape and combination.  As
other ideas, I also love the strength and simplicity of the attached logo
for a nearby little park/oasis.  I will try to find out who crafted this
logo.

**Comment by alexgleason** [comment]
Wonderful, thanks for taking a look at this @JenniferGold!

I'm open to replacing this logo with a better one, although we'd likely want to hire a professional designer for that. If you're happy enough with this logo, I think it makes sense to stick with it short of hiring a pro designer.

In my opinion it conveys our themes quite well.

* **growing food**: the plant in the center of the logo.
* **urban environment**: I think the building does a good job of conveying the urban environment that these gardens exist within.
* **people**: the building can doubly serve to convey this, since afterall buildings necessarily contain people, otherwise they're worthless. I think the plant imposed over the building offers a nice synthesis of the natural with the human-designed.

**Comment by JenniferGold** [comment]
Yes, agreed!

**Comment by alexgleason** [comment]
Awesome, thanks! I'll close this issue for now, and we'll move forward with this logo.
